bin/migrate-goldens.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+import { join, normalize, sep } from 'path';
+import { mkdir, rename } from 'node:fs/promises';
+import commandLineArgs from 'command-line-args';
+import { glob } from 'glob';
+import { stdout } from 'node:process';
+
+const { pattern = './test/**' } = commandLineArgs({ name: 'pattern', type: String, defaultOption: true }, { partial: true });
+const oldSuffix = 'screenshots/ci/golden';
+const newSuffix = 'vdiff/ci/chromium/golden';
+const dirs = await glob(`${pattern}/${oldSuffix}`, { ignore: 'node_modules/**' });
+const files = [];
+
```

Comment:
Could we list out the migration steps somewhere, either in Markdown or in comments? Just so it's super clear which migration steps this handles.

The stuff we need (so far):

1. Rename `screenshots` to `vdiff`
2. Add a new `chromium` directory inside `vdiff\ci\golden`
3. Move everything from the old `screenshots\ci\golden` into `vdiff\ci\golden`
4. "Fix' the directory/filenames to remove redundancies

For 4), I don't know if we can do it without running the tests... because [extractTestPartsFromName](https://github.com/BrightspaceUI/core/pull/3610/files#diff-1efd6ab174aa37274c8e9be049f951ecfd00b2cc15122d8aff895dd11bf3e2f4R19) uses the spaces between suites as important information that we don't have just from the filename because the old library [turned them into hyphens](https://github.com/BrightspaceUI/visual-diff/blob/main/file-helper.js#L44). 🤔 Does it therefore make sense to run this as like a `--migration` flag from the old library?